src/Data/Vector/NonEmpty.hs

```diff
@@ -216,7 +224,7 @@ newtype NonEmptyVector a = NonEmptyVector
     } deriving
       ( Eq, Ord
       , Eq1, Ord1, Show1
-      , Data, Typeable, Generic, NFData
+      , Data, Typeable, NFData
```

Comment:
per @phadej the `Generic`, `Alternative` (duh), and `Read` instances here are a big ol' lie.